components/hab/src/cli.rs

```diff
@@ -17,17 +17,18 @@ use std::{net::SocketAddr,
           result,
           str::FromStr};
 
-use crate::{hcore::{crypto::keys::PairType,
-                    package::{ident,
-                              Identifiable,
-                              PackageIdent,
-                              PackageTarget},
-                    service::{HealthCheckInterval,
-                              ServiceGroup}},
-            protocol};
+use crate::protocol;
```

Comment:
We can do something similar here:
```rust
use hab_sup_protocol as protocol;
```
If uses of `protocol::XXX` were replaced with `hab_sup_protocol::XXX`, it could just be `use hab_sup_protocol;`